suite-native/module-trading/src/hooks/general/form/useSelectedDeviceChangeEffect.ts

```diff
@@ -0,0 +1,11 @@
+import { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+
+import { selectSelectedDevice } from '@suite-common/wallet-core';
+
+export const useSelectedDeviceChangeEffect = (clearCallback: () => void) => {
```

Comment:
It does not necessarily have to be clear. wdyt?
```suggestion
export const useSelectedDeviceChangeEffect = (onDeviceChangeCallback: () => void) => {
```